packages/firebase_snippets_app/lib/snippets/firestore.dart

```diff
@@ -102,9 +104,26 @@ class FirestoreSnippets extends DocSnippet {
     // [END data_model_sub_collections]
   }
 
-  void dataBundles_loadingClientBundles() {
+  void dataBundles_loadingClientBundles() async {
     // [START data_bundles_loading_client_bundles]
-    // TODO - currently in scratch file
+    // Get a bundle from a server
+    final url = Uri.https('example.com', '/create-bundle');
+    final response = await http.get(url);
+    String string = response.body;
+    final buffer = Uint8List.fromList(string.codeUnits);
```

Comment:
nit: can we name this variable `body` instead? I think it's more descriptive than `string`.

```suggestion
    String body = response.body;
    final buffer = Uint8List.fromList(body.codeUnits);
```